src/modules/Dashboard/features/Applet/Schedule/CreateEventPopup/CreateEventPopup.tsx

```diff
@@ -16,18 +17,22 @@ export const CreateEventPopup = ({
 }: CreateEventPopupProps) => {
   const { t } = useTranslation('app');
   const eventFormRef = useRef() as RefObject<EventFormRef>;
+  const { respondentId } = useParams();
   const [currentActivityName, setCurrentActivityName] = useState('');
   const [removeAllScheduledPopupVisible, setRemoveAllScheduledPopupVisible] = useState(false);
   const [removeAlwaysAvailablePopupVisible, setRemoveAlwaysAvailablePopupVisible] = useState(false);
 
   const handleCreateEventClose = () => setCreateEventPopupVisible(false);
 
+  const isIndividualCalendar = !!respondentId;
+  const analyticsPrefix = isIndividualCalendar ? 'IC' : 'GC';
```

Comment:
could you create an enum for AnalyticsPrefix? For ex.
```
const enum AnalyticsPrefix {
   '<description for 'IC' prefix>' = 'IC',
   '<description for 'GC' prefix>' = 'GC', 
}
```

Reply:
Sure. I'll put it in the shared/consts.tsx then